src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -296,7 +296,7 @@ class FunctionBuilder[F >: Null](val parameterTypeInfo: Array[MaybeGenericTypeIn
 
       cn.accept(cw)
       bytes = cw.toByteArray
-      // CheckClassAdapter.verify(new ClassReader(bytes), false, new PrintWriter(sw1))
+      CheckClassAdapter.verify(new ClassReader(bytes), false, new PrintWriter(sw1))
```

Comment:
I think this is only useful for debugging and I found it to be INCREDIBLY slow.  I don't think it should be uncommented.

Reply:
Sorry, that was a mistake. Will fix it.

Reply:
Great!  Otherwise it looks great and I will prove.